### DISCUSSION TITLE
Problems after converting models

### DISCUSSION THREAD

**Discussion by WapMap** [upscayl/upscayl#705] [Q&A] [answered]
After converting the models I tried running them without changes and with the changes from the conversion instructions, but nothing works, I don't understand what the problem is. Models used in the conversion: https://openmodeldb.info/models/8x-NMKD-Typescale & https://openmodeldb.info/models/4x-FSDedither-Riven .

**Comment by aaronliu0130** [comment, marked as answer]
Let's try tracking this in #686 for now 🙂
